cmd/web.go

```diff
@@ -71,6 +73,33 @@ func runHTTPRedirector() {
 	}
 }
 
+func runLetsEncrypt(listenAddr, domain, directory, email string, m http.Handler) error {
+	certManager := autocert.Manager{
+		Prompt:     autocert.AcceptTOS,
```

Comment:
This HAS to be behind its own setting (`LETSENCRYPT_ACCEPT_TOS`) :scream: People will _not_ read the comments...